kvm/cpuid.go

```diff
@@ -45,6 +45,14 @@ func SetCPUID2(vcpuFd uintptr, kvmCPUID *CPUID) error {
 	return err
 }
 
+func GetCPUID2(vcpuFd uintptr, kvmCPUID *CPUID) error {
+	_, err := Ioctl(vcpuFd,
+		IIOWR(kvmGetCPUID2, 8),
```

Comment:
This can be hardcoded, because we pass the size of the area which will be read, no matter the length of the slice field. The length of the slice is passed as value inside the first 8 byte.